src/js/OTSubscriber.coffee

```diff
@@ -19,11 +19,6 @@ class TBSubscriber
     return ""
   getStyle: ->
     return {}
-  off: (event, handler) ->
-    return @
-  on: (event, handler) ->
```

Comment:
same as above, why is this being deleted?

Reply:
This has changed, dont know why its not outdated.